gtsfm/frontend/matcher/matcher_base.py

```diff
@@ -26,21 +28,25 @@ class MatcherBase(metaclass=abc.ABCMeta):
     @abc.abstractmethod
     def match(
         self,
-        descriptors_im1: np.ndarray,
-        descriptors_im2: np.ndarray,
+        keypoints_i1: Keypoints,
```

Comment:
could we explain in the docstring why we need the keypoints, as well as descriptors? Explaining that SuperGlue uses "positional encoding" of the keypoints, as well as descriptors?

Reply:
Do we want to add this? Because some other matcher might not use it as positional encoding?

Reply:
can we just add a brief note explaining why it might be used? 
```python
# Some matcher implementations (such as SuperGlue) utilize keypoint coordinates as
# positional encoding, so our matcher API provides them for optional use.
```

Reply:
done

Reply:
added